packages/react-components/src/components/AttachmentUploadCards.tsx

```diff
@@ -113,11 +101,7 @@ export const _AttachmentUploadCards = (props: FileUploadCardsProps): JSX.Element
               progress={attachment.progress}
               key={attachment.id}
               attachmentExtension={extension(attachment.filename)}
-              actionIcon={
-                <IconButton className={iconButtonClassName} ariaLabel={removeAttachmentButtonString()}>
-                  <Icon iconName="CancelFileUpload" style={actionIconStyle} />
-                </IconButton>
-              }
+              actionIcon={<Icon iconName="CancelFileUpload" style={actionIconStyle} />}
```

Comment:
It usually is a bad idea to use `styles` prop because they are calculated on every render. Is it possible to use `className` prop here instead of `styles` one?